platform-api/src/routers/forms.py

```
@@ -0,0 +1,164 @@
+        # Log the exception with context, but don't break submission flow
+        import httpx
+        if isinstance(e, httpx.HTTPStatusError):
+            logger.exception(
+                "Discord notification failed for webhook host %s, request_id=%s",
+                webhook_host, request_id
+            )
+        elif isinstance(e, httpx.HTTPError):
+            logger.exception(
+                "Discord notification failed for webhook host %s, request_id=%s",
+                webhook_host, request_id
+            )
```

Ruff (0.15.11)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)